source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/__tests__/filtering-by-organization-acceptance-test.ts

```diff
@@ -0,0 +1,96 @@
+import { describe, it, beforeEach } from '@bigtest/mocha';
+import { visit, location } from '@bigtest/react';
+import { expect } from 'chai';
+
+import {
+  setupApplicationTest,
+  setupRequestInterceptor,
+  useFakeAuthentication,
+  wait
+} from 'tests/helpers/index';
+
+import page from './page';
+import { threeProjects, DTProjects } from './scenarios';
+
+describe('Acceptance | Project Directory | Filtering | By Organization', () => {
+  setupApplicationTest();
+  setupRequestInterceptor();
+  useFakeAuthentication();
+
+  beforeEach(function () {
+    const { server } = this.polly;
+
+    server.namespace('/api', () => {
+      server.get('/projects').intercept((req, res) => {
+        res.status(200);
+        res.headers['Content-Type'] = 'application/vnd.api+json';
+
+        if (!req.query['filter[organization-header]']) {
+          res.json(threeProjects);
+
+        } else {
+          res.json(DTProjects);
+        }
+
+        expect(req.headers.Organization).to.equal(undefined);
+      });
+    });
+  });
+
+
+  describe('navigating to the project directory page', () => {
+    beforeEach(async function() {
+      await visit('/directory');
+    });
+
+    it('is on the directory page', () => {
+      expect(location().pathname).to.equal('/directory');
+    });
+
+    it('shows all the possible projects', () => {
+      const rows = page.table.rows();
+
+      expect(rows.length).to.equal(3);
```

Comment:
Perhaps not necessary but shouldn't we test that we are actually render the project and not just if exists?

Reply:
that's fair. I'll add